test/test_tensorboard.py

```diff
@@ -0,0 +1,454 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from common_utils import run_tests
+
+from torch.utils.tensorboard import x2num, SummaryWriter
+import torch
+import numpy as np
+import unittest
+from torch.utils.tensorboard import summary
+from torch.utils.tensorboard.utils import _prepare_video, convert_to_HWC
+import pytest
```

Comment:
pytest is not an existing dependency of the test suite. I see you have only used it for `pytest.raises`; in this case, you should just use the built in `assertRaises` instead